odpf/optimus/runtime_service.proto

```diff
@@ -750,3 +761,34 @@ message BackupDryRunRequest {
 message BackupDryRunResponse {
     repeated string resource_name = 1;
 }
+
+message BackupRequest {
+    string project_name = 1;
+    string datastore_name = 2;
+    string resource_name = 3;
+    string namespace = 4;
+    string description = 5;
+    bool ignore_downstream = 6;
+    map<string, string> config = 7;
+}
+
+message BackupResponse {
+    repeated string urn = 1;
+}
+
+message ListBackupsRequest {
+    string project_name = 1;
+    string datastore_name = 2;
+    string namespace = 3;
+}
+
+message ListBackupsResponse {
+    repeated BackupSpec backup_list = 1;
```

Comment:
I think we can call this `backups` instead of backup_list.

Reply:
agree.